src/test/functional/features/marriage-broken-down.feature

```diff
@@ -0,0 +1,24 @@
+Feature: Marriage Broken Down
```

Comment:
This should be agnostic e.g.
```suggestion
Feature: Partnership Broken Down
```

Reply:
maybe this should be a separate change as it doesn't relate to this PR? Also, the filename will need to be updated